src/routes/api.php

```diff
@@ -36,6 +37,7 @@
     Route::patch('/formats/{format}', [FormatController::class, 'partialUpdate']);
     Route::apiResource('meetings', MeetingController::class);
     Route::patch('/meetings/{meeting}', [MeetingController::class, 'partialUpdate']);
+    Route::apiResource('meetings.changes', MeetingChangeController::class)->only([ 'index' ]);
```

Comment:
Do you need the `->only(['index'])`? Is it smart enough to figure out that the controller has only defined the `index` method itself? I'm good either way - just curious.

Reply:
I checked the docs and it said that  Laravel automatically registers the standard set of routes for an API resource but it will NOT automatically restrict the routes based on which methods you have implemented in your controller